lib/configure.js

```diff
@@ -269,6 +269,22 @@ function configure (gyp, argv, callback) {
         log.error(logprefix, 'Could not find exports file')
         return callback(new Error(msg))
       }
+    } else if (process.platform === 'os390') {
+      var node_root_dir = findNodeDirectory()
+      var candidates = ['include/node/node.x',
+                        'out/Release/node.x',
+                        'out/Debug/node.x',
+                        'node.x']
+      for (var next = 0; next < candidates.length; next++) {
+         node_exp_file = path.resolve(node_root_dir, candidates[next])
+         try {
+           fs.accessSync(node_exp_file, fs.R_OK)
```

Comment:
`fs.accessSync()` does not exist in v0.10 and v0.12 (see also #955) but since this is a z/OS-only code path, I guess that's alright.  That said, can't this be merged with the AIX code path above?

Reply:
Yes it can be merged with AIX code. Will do